docs/91_version_history.rst

```diff
@@ -34,67 +34,35 @@ Release Date: 2019-08-15
        - Addition of support for hybrid plugins. A hybrid plugin is a JSON file that defines another plugin to load along with some default configuration for that plugin. This gives a means to create a new plugin by customising the configuration of an existing plugin. An example might be a plugin for a specific modbus device type that uses the generic modbus plugin and a predefined modbus map.
-       - Support  for reading floating point values in a pair of 16 bit registers has been added to the modbus plugin.
        - The notification service has been improved to allow the re-trigger time of a notification to be defined by the user on a per notification basis.\n\n
        - A new environment variable, FOGLAMP_PLUGIN_PATH has been added to allow plugins to be stored in multiple locations or locations outside of the usual FogLAMP installation directory.
        - Added support for FOGLAMP_PLUGIN_PATH environment variable, that would be used for searching additional directory paths for plugins/filters to use with FogLamp.
        - FogLAMP packages for the Google Coral Edge TPU development board have been made available.
        - Support has been added to the OMF north plugin for the PI Web API OMF endpoint. The PI Server functionality to support this is currently in beta test.
-       - The GUI now support password type configuration items. This is used to allow the user to enter passwords or keys and keep the values hidden from onlookers. The data entered is replaced with ‘*' characters in the input field as the user types and will be displayed as '*’ characters when retrieved.
 
+    - Bug Fix/Improvements:
 
-    - Bug Fix:
-
-       - The filter plugin python35 crashed on the Buster release on the Rasperberry Pi, this has now been resolved.
-       - The FFT filter now enforces that the number of samples must be a power of 2.
        - An issue with the notification service becoming unresponsive on the Raspberry Pi Buster release has been resolved.
-       - The change filter has been made available on CentOS and RedHat 7 releases.
-       - The ThingSpeak north plugin was not updated in line with changes to the timestamp handling in FogLAMP, this resulted in a crash when it tried to send data to ThingSpeak. This has been resolved and the cause of the crash also fixed such that now an error will be logged rather than the task crashing.
        - A debug message was being incorrectly logged as an error when adding a Python south plugin. The message level has now been corrected.
-       - The notification service would sometimes become unresponsive after calling the notify-python35 plugin, this has now been resolved.
-       - The phidget south plugin reconfiguration method would crash the service on occasions, this has now been resolved.
        - A fatal error would be logged on the Buster release for the Raspberry Pi by the notification service and it would become unresponsive. This is now resolved.
```

Comment:
this is a duplicate, we already have `- An issue with the notification service becoming unresponsive on the Raspberry Pi Buster release has been resolved.` above